index.html

```diff
@@ -394,49 +395,49 @@ <h4>LET'S CONNECT</h4>
       <div class="overlay">
         <!-- <img src="images/contactformshapes.svg" alt="footershapes" /> -->
 
-        <div class="content"> 
-        <article class="footer-text">
-          <h2>Contact me</h2>
-          <p>
-            If you have an application you are interested in developing, a
-            feature that you need built or a project that needs coding. I’d love
-            to help with it
-          </p>
-        </article>
+        <div class="content">
+          <article class="footer-text">
+            <h2>Contact me</h2>
+            <p>
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+          </article>
+
+          <form action="https://formspree.io/f/xpzbopev" method="post">
+            <div class="name">
+              <input
+                type="text"
+                name="text"
+                required
+                placeholder="Yeremias"
+                maxlength="30"
```

Comment:
- [ ] There is a problem with resizing, The contact form is not available for mobile screens, Please make sure to fix it

### See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/166641022-51cc506a-d66d-4f6e-bae7-d2c67bb4ac58.png)